json-guides/microprofile-config.json

```diff
@@ -12,58 +12,129 @@
             "guide_duration" : "25 minutes",
             "description": [
               "temp"
+            ],
+            "sections" : [
+                {
+                  "name": "backgroundConcepts",
+                  "title": "Background concepts",
+                  "description": [
+                    "temp"
+                  ]
+                }
             ]
         },
         {
-            "name": "ConfigureViaInject",
-            "title": "Configuring with Default Injection Value",
+            "name": "ExampleScenario",
+            "title": "Example: xxx",
             "description": [
-                "Microprofile-Config provides ways to inject configured values into your beans using the <code>@Inject</code> and the <code>@ConfigProperty</code> qualifier"
+                "temp"
+            ]
+        },
+        {
+            "name": "AddLibertyMPFaultTolerance",
+            "title": "Enabling MicroProfile Config",
+            "description": [
+                "Begin by enabling the MicroProfile Config feature in your <code>server.xml</code> file."
             ],
             "instruction": [
-                "Add the <code>@Inject @ConfigProperty(name=\"download-url\", defaultValue=\"ftp://music.com/us/download\")</code> on line 9, after the <code>private Config config</code> or click <action title='Configure inject using ConfigProperty' onclick=\"microprofileConfigCallBack.addConfigInjectButton(event)\"><b>@Inject @ConfigProperty(name=\"download-url\", defaultValue=\"ftp://music.com/us/download\")</b></action>. Then, click <action title='Save' onclick=\"microprofileConfigCallBack.saveButton(event)\"><b>Save</b></action> on the editor menu pane."
+              "Add <b>&lt;feature>mpConfig-1.1&lt;/feature>."
+            ]
+        },
+        {
+            "name": "ConfigureViaInject",
+            "title": "Injecting config properties through annotations",
```

Comment:
I think it is better to use "Configuring configuration properties through injections" but I could be wrong.

Reply:
I'll change to 'Injecting configuration properties through annotations' and we will discuss more as a team.